tests/test_ble_lifecycle_receive_targets.py

```
@@ -0,0 +1,1645 @@
+    BLEReceiveRecoveryService._log_empty_read_warning(iface)
+    iface._last_empty_read_warning = 10**9
+    BLEReceiveRecoveryService._log_empty_read_warning(iface)
+
+    iface._state_manager = importlib.import_module(
+        "meshtastic.interfaces.ble.state"
+    ).BLEStateManager()
+    iface._shutdown_event = threading.Event()
+    iface.close()
+
+
+def test_lifecycle_and_receive_remaining_branch_targets(
```

Ruff (0.15.6)

[warning] Too many statements (195 > 50)

(PLR0915)